invenio_rdm_records/assets/semantic-ui/js/invenio_rdm_records/src/deposit/api/DepositApiClient.js

```diff
@@ -322,10 +348,27 @@ export class DepositFileApiClient {
  * Default File API Client for deposits.
  */
 export class RDMDepositFileApiClient extends DepositFileApiClient {
-  initializeFileUpload(initializeUploadUrl, filename) {
+  constructor(additionalApiConfig, defaultTransferType, transferTypes) {
+    super(additionalApiConfig);
+    this.defaultTransferType = defaultTransferType || "L";
+    this.transferTypes = transferTypes || ["L"];
```

Comment:
Is there a possibility to have at the same instance multiple transfer types configured? If not, should we convert this to a single value?

Reply:
There is a possibility for one uploader instance to support many types. Currently `RDMUppyUploaderPlugin` does `M` and `L`, but we could potentially add plugins to the instance, supporting also Fetch (`F`) or Remote (`R`), e.g. by adaptation of some [Source plugins](https://uppy.io/docs/url/)